Library/Formula/procyon-decompiler.rb

```diff
@@ -0,0 +1,27 @@
+class ProcyonDecompiler < Formula
+  desc "Procyon Java Decompiler is part of the Procyon suite of Java metaprogramming tools focused on code generation and analysis."
```

Comment:
At the moment we're [only](https://github.com/Homebrew/homebrew/blob/master/Library/Homebrew/cmd/audit.rb#L313) auditing `desc` length during `brew audit --strict`.